components/apps_modal/dialog_introduction_text.jsx

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as Markdown from 'utils/markdown';
+import {getSiteURL} from 'utils/url';
+
+export default class DialogIntroductionText extends React.PureComponent {
```

Comment:
Why not typescript?

Reply:
I copied the file verbatim and applied no changes. Will make the file TS :+1: